<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Real Estate CRM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>917</width>
        <height>586</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Agents">
       <property name="maximumSize">
        <size>
         <width>774</width>
         <height>491</height>
        </size>
       </property>
       <attribute name="title">
        <string>Agents</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="agentWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="agentSearchEdit">
                <property name="placeholderText">
                 <string>Search...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="agentSearchButton">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="addAgent">
                <property name="text">
                 <string>Add Agent</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="editAgent">
                <property name="text">
                 <string>Edit Agent</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteAgent">
                <property name="text">
                 <string>Delete Agent</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="agentsTable">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
             </property>
             <property name="gridStyle">
              <enum>Qt::PenStyle::SolidLine</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>10</number>
             </property>
             <property name="columnCount">
              <number>7</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>First Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Last Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Phone</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Email</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Start Date </string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>End Date</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Clients">
       <property name="maximumSize">
        <size>
         <width>797</width>
         <height>534</height>
        </size>
       </property>
       <attribute name="title">
        <string>Clients</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="clientSearchEdit">
                <property name="placeholderText">
                 <string>Search...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clientSearchButton">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="addClient">
                <property name="text">
                 <string>Add Client</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="editClient">
                <property name="text">
                 <string>Edit Client</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteClient">
                <property name="text">
                 <string>Delete Client</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="clientsTable">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>10</number>
             </property>
             <property name="columnCount">
              <number>8</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>First Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Last Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Phone</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Email</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Married</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Budget</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>BudgetType</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Properties">
       <property name="maximumSize">
        <size>
         <width>797</width>
         <height>534</height>
        </size>
       </property>
       <attribute name="title">
        <string>Properties</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLineEdit" name="propertySearchEdit">
                <property name="placeholderText">
                 <string>Search...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="propertySearchButton">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_6">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="addProperty">
                <property name="text">
                 <string>Add Property</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="editProperty">
                <property name="text">
                 <string>Edit Property</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteProperty">
                <property name="text">
                 <string>Delete Property</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="propertiesTable">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>10</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Size</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Price</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Bedrooms</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Bathrooms</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Location</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Available</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Listing Type</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Contracts">
       <property name="maximumSize">
        <size>
         <width>797</width>
         <height>534</height>
        </size>
       </property>
       <attribute name="title">
        <string>Contracts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QFrame" name="frame_7">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLineEdit" name="contractSearchEdit">
                <property name="placeholderText">
                 <string>Search...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="contractSearchButton">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_8">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="addContract">
                <property name="text">
                 <string>Add Contract</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="editContract">
                <property name="text">
                 <string>Edit Contract</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteContract">
                <property name="text">
                 <string>Delete Contract</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="contractsTable">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>10</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Property ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Client ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Agent ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Price</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Start Date</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>End Date</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Active</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
